composer.json

```diff
@@ -42,9 +42,19 @@
             "**/Tests/", "**/spec/"
         ]
     },
+    "repositories": [
```

Comment:
For now it need to stay. We don't control sylius repos, and without it we can't use aliases.